app/page/template/wire-main.htm

```diff
@@ -5,18 +5,23 @@
      data-uie-value="is-loading">
   <!-- ko stopBinding: true -->
     <div id="app" class="app">
-      <div id="left-column" class="left-column">
-        #include('list/background.htm')
+      <div id="left-column" class="left-column" data-bind="css: {'left-column--white': isActivatedAccount}">
```

Comment:
Maybe we should actually make this part of the background.htm and move the location back